src/applications/form-renderer/_config/formConfig.js

```diff
@@ -181,3 +181,31 @@ export const formConfig2 = {
     },
   },
 };
+
+// This is a sample of the data structure produced by content-build.
+export const normalizedForm = {
+  id: 71160,
```

Comment:
**QUESTION**
I thought we settled on `cmsId` here. Has that been changed?

It is interesting to me to have `cmsId` on the node but `id` on the chapters/steps/other things. Perhaps we need to iron this out a bit still.

Reply:
We absolutely did, and `cmsId` appears on https://pr18811-ps4nwwul37jtyembecv4bg0gafmyl3oj.ci.cms.va.gov/data/cms/digital-forms.json. Guess I wrote this before we had that discussion. This is the downside to using mock data for testing, but doesn't actually impact anything at the moment, because I don't think we're using `cmsId` for anything on the front-end yet. Nonetheless, I'll update the mock form to reflect what is coming from `content-build`.

Reply:
> It is interesting to me to have cmsId on the node but id on the chapters/steps/other things. Perhaps we need to iron this out a bit still.

I felt no immediate need to disambiguate it as there are no other IDs on the chapter level competing for attention, but I could be persuaded to change the key label to `chapterId`, `pageId`, or `stepId`.